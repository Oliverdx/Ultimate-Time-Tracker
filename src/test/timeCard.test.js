import React from 'react';
import renderer from 'react-test-renderer';
import TimeCard from '../components/timeCard';

describe('TimeCard component test', () => {

  const time = {
    id: 0,
    day: '2021-01-01',
    start: '',
    exit: '',
    lunchexit: '',
    lunchreturn: ''
  }

  it('Should render correctly', () => {
    const component = renderer.create(<TimeCard time={time}/>).toJSON();
    expect(component).toMatchSnapshot();
  });
    
});