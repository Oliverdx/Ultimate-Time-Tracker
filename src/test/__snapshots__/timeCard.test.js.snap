// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimeCard component test Should render correctly 1`] = `
<div>
  <div>
    <p>
      01
    </p>
    <span>
      Janeiro
    </span>
  </div>
  <div>
    <div>
      <span>
        Início
      </span>
      <p
        className=""
      >
        Invalid Date
      </p>
    </div>
    <div>
      <span>
        Almoço
      </span>
      <p
        className=""
      >
        Invalid Date
      </p>
    </div>
    <div>
      <span>
        Retorno
      </span>
      <p
        className=""
      >
        Invalid Date
      </p>
    </div>
    <div>
      <span>
        Saída
      </span>
      <p
        className=""
      >
        Invalid Date
      </p>
    </div>
    <div>
      <span>
        Trabalhado
      </span>
      <p
        className=" "
      >
        NaN:NaN
      </p>
    </div>
  </div>
</div>
`;
